editor/property_selector.cpp

```diff
@@ -338,52 +338,31 @@ void PropertySelector::_item_selected() {
 	}
 	String name = item->get_metadata(0);
 
-	String class_type;
-	if (type != Variant::NIL) {
-		class_type = Variant::get_type_name(type);
-
-	} else {
-		class_type = base_type;
-	}
-
 	DocData *dd = EditorHelp::get_doc_data();
 	String text;
 
+	Map<String, DocData::ClassDoc>::Element *E = dd->class_list.front();
```

Comment:
I'd put this in the `for` initializer, it doesn't need to be in the outer scope.

Reply:
updated. i could also put braces around the if-else if that is desirable. i would like to get this in. let me know.